src/components/__tests__/navigation/MobileNav.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@/utils/test-utils';
+import MobileNav from '@/components/navigation/MobileNav';
+
+describe('MobileNav component', () => {
+  test('matches the snapshot', () => {
+    const { asFragment } = render(
+      <MobileNav isOpen={false} isVisible={true} handleItemClick={jest.fn()} />
```

Comment:
this seems odd - why there is prop isOpen and isVisible ? Dont they overlap in their function ?